packages/lib-classifier/src/plugins/drawingTools/components/RotateHandle/RotateHandle.js

```diff
@@ -9,7 +9,11 @@ const RotateHandle = forwardRef(function RotateHandle(
   const transform = `translate(${x}, ${y}) scale(${1 / scale})`
 
   return (
-    <g ref={ref} transform={transform}>
+    <g
+      ref={ref}
+      transform={transform}
+      data-testid='rotate-handle'
+    >
       <svg
         xmlns='http://www.w3.org/2000/svg'
```

Comment:
You can add alt text to SVGs by giving them `role=“img”` and an `aria-label`.

Reply:
If you do do that, I recommend [Contextually Marking up accessible images and SVGs](https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html) by Scott O’Hara.